cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -750,63 +869,68 @@ systemStatsW p =
     cpuUsageBar = updateAttrMap
                   (A.mapAttrNames [ (cpuDoneAttr, P.progressCompleteAttr)
                                   , (cpuToDoAttr, P.progressIncompleteAttr)
                                   ]
                   ) $ bar cpuLabel (lvsCPUUsagePerc p)
-    cpuLabel = Just $ take 5 (show $ lvsCPUUsagePerc p * 100) ++ "%"
+    cpuLabel = Just $ withOneDecimal (lvsCPUUsagePerc p * 100) ++ "%"
 
     diskUsageRBar :: forall n. Widget n
     diskUsageRBar = updateAttrMap
                     (A.mapAttrNames [ (diskIODoneAttr, P.progressCompleteAttr)
                                     , (diskIOToDoAttr, P.progressIncompleteAttr)
                                     ]
                     ) $ bar diskUsageRLabel (lvsDiskUsageRPerc p)
-    diskUsageRLabel = Just $ take 5 (show $ lvsDiskUsageRCurr p) ++ " KB/s"
+    diskUsageRLabel = Just $ withOneDecimal (lvsDiskUsageRCurr p) ++ " KB/s"
 
     diskUsageWBar :: forall n. Widget n
     diskUsageWBar = updateAttrMap
                     (A.mapAttrNames [ (diskIODoneAttr, P.progressCompleteAttr)
                                     , (diskIOToDoAttr, P.progressIncompleteAttr)
                                     ]
                     ) $ bar diskUsageWLabel (lvsDiskUsageWPerc p)
-    diskUsageWLabel = Just $ take 5 (show $ lvsDiskUsageWCurr p) ++ " KB/s"
+    diskUsageWLabel = Just $ withOneDecimal (lvsDiskUsageWCurr p) ++ " KB/s"
 
     networkUsageInBar :: forall n. Widget n
     networkUsageInBar = updateAttrMap
                         (A.mapAttrNames [ (networkIODoneAttr, P.progressCompleteAttr)
                                         , (networkIOToDoAttr, P.progressIncompleteAttr)
                                         ]
                         ) $ bar networkUsageInLabel (lvsNetworkUsageInPerc p)
-    networkUsageInLabel = Just $ take 5 (show $ lvsNetworkUsageInCurr p) ++ " KB/s"
+    networkUsageInLabel = Just $ withOneDecimal (lvsNetworkUsageInCurr p) ++ " KB/s"
 
     networkUsageOutBar :: forall n. Widget n
     networkUsageOutBar = updateAttrMap
                          (A.mapAttrNames [ (networkIODoneAttr, P.progressCompleteAttr)
                                          , (networkIOToDoAttr, P.progressIncompleteAttr)
                                          ]
                          ) $ bar networkUsageOutLabel (lvsNetworkUsageOutPerc p)
-    networkUsageOutLabel = Just $ take 5 (show $ lvsNetworkUsageOutCurr p) ++ " KB/s"
+    networkUsageOutLabel = Just $ withOneDecimal (lvsNetworkUsageOutCurr p) ++ " KB/s"
 
     bar :: forall n. Maybe String -> Float -> Widget n
-    bar lbl pcntg = P.progressBar lbl pcntg
+    bar = P.progressBar
     lvsMemUsagePerc = lvsMemoryUsageCurr p / max 200 (lvsMemoryUsageMax p)
 
-nodeInfoW :: LiveViewState a -> Widget ()
+withOneDecimal :: (Fractional f, Ord f, Show f) => f -> String
+withOneDecimal x
+  | x < 0.1 = "0.0"
+  | otherwise = uncurry (<>) . (id *** take 2) $ span (/= '.') (show x)
```

Comment:
this seems to be the same basic functionality as those in `Numeric` ie `ShowFFloat`

Reply:
Done, thanks!